.travis.yml

```diff
@@ -4,7 +4,11 @@ python:
 
 addons:
   apt:
+    sources:
+      - ubuntu-toolchain-r-test
+      - llvm-toolchain-trusty-5.0
```

Comment:
`llvm` **and** `g++-6`?

Reply:
I copied from prjxray, I'll see if "ubuntu-toolchain-r-test" works.